watcher/local/watcher.go

```diff
@@ -25,6 +26,12 @@ import (
 	"perun.network/go-perun/watcher"
 )
 
+const (
+	// Duration for which the watcher will wait for latest transactions
+	// when an adjudicator event is received.
+	statesFromClientWaitTime = 1 * time.Millisecond
```

Comment:
I think this should be configurable, what do you think @matthiasgeihs ?  
It would be already good for debugging when using it in deployment.

Reply:
Currently, this delay is used only to account for receiving events that are already published, but not read by the watcher. That is also the reason, why this timeout is very small.

However, as we had discussed in the call, in future we could introduce the concept of `Sync interval` and once we do that, we will have to make this as a configurable parameter of the watcher.